### PR TITLE
Use PID control on shooter

### LINKED ISSUE #3: Setup PID control in Shooter Subsystem and on SparkMAX

### PR DESCRIPTION
**Pull request by toriawlam**

### CODE REVIEW

**Review by AndrewLester — APPROVED** [review]
Looks good

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 1]
```diff
@@ -31,9 +40,12 @@ public void lowerPiston() {
...
-    /** @param shooterSpeed Velocity of shooter motors in RPM */
     public void shoot(double shooterSpeed) {
-        // shooterController.setReference(shooterSpeed, ControlType.kVelocity);
-        shooterMotor1.set(shooterSpeed);
+        shooterController.setReference(2000, ControlType.kVelocity);
+        // shooterMotor1.set(shooterSpeed);
+    }
+
+    public void periodic() {
+        System.out.println(this.shooterEncoder.getVelocity());
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:52]: Perhaps networktables would be a better way to get this info for debugging?

**Review by ArchdukeTim — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 2]
```diff
@@ -31,9 +40,12 @@ public void lowerPiston() {
         shooterSolenoid.set(false);
     }
 
-    /** @param shooterSpeed Velocity of shooter motors in RPM */
     public void shoot(double shooterSpeed) {
-        // shooterController.setReference(shooterSpeed, ControlType.kVelocity);
-        shooterMotor1.set(shooterSpeed);
+        shooterController.setReference(2000, ControlType.kVelocity);
+        // shooterMotor1.set(shooterSpeed);
+    }
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java]: Maybe have two methods, one that takes  a speed and one that doesn't so you can use velocity vs voltage to control

Comment (AndrewLester) [reply src/main/java/frc/robot/subsystems/ShooterSubsystem.java:46]: Indeed. We'll probably make this change next time. I wish there was a `tunable` for Java wpilib...

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java (hunk 1 above)
Comment (AndrewLester) [reply src/main/java/frc/robot/subsystems/ShooterSubsystem.java:52]: Indeed. We'll probably make this change next time. I wish there was a `tunable` for Java wpilib...

**Pull request merged**